src/zac/demo/models.py

```diff
@@ -19,8 +19,15 @@ class SiteConfiguration(SingletonModel):
         _('Secret'), max_length=512, blank=True,
         help_text=_('Dit Secret wordt voor alle APIs gebruikt, tenzij een '
                     'API specifiek Secret is opgegeven'))
-    google_api_key = models.CharField(
-        _('Google API-key'), max_length=255, blank=True)
+    google_maps_api_key = models.CharField(
+        _('Google maps API-key'), max_length=255, blank=True)
+    google_maps_lat = models.DecimalField(
+        _('Google maps latitude'), max_digits=9, decimal_places=6, blank=True,
+        default='52.36991896035238')
```

Comment:
hrm, you specify `decimal_places=6` and provide a value with 14 decimal places. If we don't need that precision, it would make a bit more sense to truncate the default to 6 decimal places